include/openrave/environment.h

```diff
@@ -30,7 +30,7 @@ namespace OpenRAVE {
 #if OPENRAVE_STD_SCOPED_LOCK
 #include <mutex>
 using EnvironmentMutex = ::std::recursive_mutex;
-using EnvironmentLock  = ::std::scoped_lock<std::recursive_mutex>;
+using EnvironmentLock  = ::std::unique_lock<std::recursive_mutex>;
```

Comment:
sorry about this, commented https://github.com/rdiankov/openrave/pull/1120/files#r1882322482

Reply:
I don't know what is the expected behavior - I would even remove boost path here.

Reply:
Let's make sure there are no regressions here with changing to unique_lock. Thanks